pkg/cmdutil/shell/shell.go

```diff
@@ -0,0 +1,85 @@
+package shell
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"os/exec"
+
+	"github.com/phuslu/log"
+)
+
+func MoveFile(source string, destination string) error {
```

Comment:
Isn't this file a copy of os/util.go?

Reply:
Yep, I renamed it to avoid the namespace clash, removed!

Reply:
Have you pushed it?
I still can see the duplicated file

Reply:
I have now